### ISSUE TITLE
Search fuzziness

### ISSUE THREAD

**Issue by edgartdata** [ycba-cia/blacklight-collections2#204]
@yulgit1 When I search for 'falcon' http://10.5.96.187:3000/catalog/tms:1193 does not appear although it has the subject tag of 'falcons (birds)'. I would think we would want the search mechanism to be inclusive? Or are we concerned about providing too many results if we err on the side of fuzziness? @flapka? others?

**Comment by yulgit1** [comment]
@edgardata - to accommodate this I think indexing *_txt fields must use the PorterStemFilterFactory

as explained here:

https://stackoverflow.com/questions/38511261/solr-how-to-match-singular-and-plural-words

**Comment by yulgit1** [comment]
reconfigured solo as : https://github.com/yulgit1/ycba-ansible1/commit/6b8cac9647b74f13a49944c879b4fa917baae51b

indexed one record : tms:1193, a search on 'falcon' should now find that record

full indexing this weekend

**Comment by edgartdata** [comment]
Yes it does! tms:1193 is the _Study of Birds_ by an unknown 18th century artist below (second image from the left), which only has "falcons (birds)" as a subject tag in its record.
![fuzzy search results](https://user-images.githubusercontent.com/4093644/83282640-e44bba80-a1a7-11ea-9b94-3baf4e265786.png)

**Comment by edgartdata** [comment]
All: let's test the search fuzziness next week and discuss results next Friday June 5.

**Comment by edgartdata** [comment]
boat -->boating?

**Comment by flapka** [comment]
(For post launch:) 

Would it be possible to implement an *exact* search feature, i.e. to turn off automatic stemming with use of a syntax such as quotation marks? 

If, for example, a researcher is interested in the concept *billing*, a search on that term will yield lots of false hits -- mostly records containing the name *Bill*. 

In Google, and in a lot of online catalogs, exact searches are implemented via quotation marks.
